frontend/src/App.tsx

```diff
@@ -117,15 +117,18 @@ function RouteSwitcher() {
     <Switch>
       {Object.values(ROUTES).concat(Object.values(routes))
         .map((route, index) =>
-          <AuthRoute
-            key={index}
-            path={getRoutePath(route)}
-            sidebar={route.sidebar}
-            requiresAuth={!route.noAuthRequired}
-            requiresCluster={!route.noCluster}
-            exact={!!route.exact}
-            children={<route.component />}
-          />
+          route.name === 'OidcAuth' ?
+            <Route path={route.path} component={() => <route.component/>} key={index}/>
+            :
```

Comment:
The route you configured should be used here. I don't understand why you need this distinction.